specs/mrtcore/PrimaryLanguageOverride/PrimaryLanguageOverride.md

```diff
@@ -0,0 +1,121 @@
+# ApplicationLanguages.PrimaryLanguageOverride
+
+This feature provides the ability to change the application language for WindowsAppSDK applications,
+comparable to the one in
+[Windows.Globalization.ApplicationLanguages.PrimaryLanguageOverride](https://learn.microsoft.com/uwp/api/windows.globalization.applicationlanguages).
+The benefit of this addition is that it applies to both packaged and un-packaged WindowsAppSDK
+applications, whereas the existing one is not supported for un-packaged.
+
+# Background
+
+Changing the application language using
+[PrimaryLanguageOverride](https://learn.microsoft.com/uwp/api/windows.globalization.applicationlanguages.primarylanguageoverride)
+property from `Windows.Globalization.ApplicationLanguages` namespace is not supported for
+WindowsAppSDK un-packaged apps, only for packaged. To support the same behavior for un-packaged
+applications `PrimaryLanguageOverride` property is introduced in
+`Microsoft.Windows.ApplicationModel.Resources` MRTCore namespace. This property
+(`Microsoft.Windows.ApplicationModel.Resources.PrimaryLanguageOverride`) overrides the behavior of
+the existing one (`Windows.Globalization.ApplicationLanguages.PrimaryLanguageOverride`).
```

Comment:
"FooOverride overrides the behavior of BarOverride" is too many instances of the word 'override'! 

Maybe:
> This property
(`Microsoft.Windows.ApplicationModel.Resources.PrimaryLanguageOverride`) **takes precedence over**
the existing one (`Windows.Globalization.ApplicationLanguages.PrimaryLanguageOverride`).

Reply:
sounds good. Updated